benchmark/tune.json

```diff
@@ -0,0 +1 @@
arkGroup",{"data":{"256bit":["BenchmarkGroup",{"data":{"Real1D":["BenchmarkGroup",{"data":{},"tags":[]}],"Mom1D":["Parameters",{"gctrial":true,"time_tolerance":0.05,"samples":10000,"evals":40,"gcsample":false,"seconds":5.0,"overhead":0.0,"memory_tolerance":0.01}]},"tags":[]}],"128bit":["BenchmarkGroup",{"data":{"Real1D":["BenchmarkGroup",{"data":{},"tags":[]}],"Mom1D":["Parameters",{"gctrial":true,"time_tolerance":0.05,"samples":10000,"evals":383,"gcsample":false,"seconds":5.0,"overhead":0.0,"memory_tolerance":0.01}]},"tags":[]}],"64bit":["BenchmarkGroup",{"data":{"Real1D":["BenchmarkGroup",{"data":{},"tags":[]}],"Mom1D":["Parameters",{"gctrial":true,"time_tolerance":0.05,"samples":10000,"evals":905,"gcsample":false,"seconds":5.0,"overhead":0.0,"memory_tolerance":0.01}]},"tags":[]}],"16bit":["BenchmarkGroup",{"data":{"Real1D":["BenchmarkGroup",{"data":{},"tags":[]}],"Mom1D":["Parameters",{"gctrial":true,"time_tolerance":0.05,"samples":10000,"evals":994,"gcsample":false,"seconds":5.0,"overhead":0.0,"memory_tolerance":0.01}]},"tags":[]}]},"tags":[]}],"get_offdiagonal":["BenchmarkGroup",{"data":{"256bit":["BenchmarkGroup",{"data":{"Real1D":["Parameters",{"gctrial":true,"time_tolerance":0.05,"samples":10000,"evals":975,"gcsample":false,"seconds":5.0,"overhead":0.0,"memory_tolerance":0.01}],"Mom1D":["Parameters",{"gctrial":true,"time_tolerance":0.05,"samples":10000,"evals":7,"gcsample":false,"seconds":5.0,"overhead":0.0,"memory_tolerance":0.01}]},"tags":[]}],"128bit":["BenchmarkGroup",{"data":{"Real1D":["Parameters",{"gctrial":true,"time_tolerance":0.05,"samples":10000,"evals":989,"gcsample":false,"seconds":5.0,"overhead":0.0,"memory_tolerance":0.01}],"Mom1D":["Parameters",{"gctrial":true,"time_tolerance":0.05,"samples":10000,"evals":10,"gcsample":false,"seconds":5.0,"overhead":0.0,"memory_tolerance":0.01}]},"tags":[]}],"64bit":["BenchmarkGroup",{"data":{"Real1D":["Parameters",{"gctrial":true,"time_tolerance":0.05,"samples":10000,"evals":996,"gcsample":false,"seconds":5.0,"overhead":0.0,"memory_tolerance":0.01}],"Mom1D":["Parameters",{"gctrial":true,"time_tolerance":0.05,"samples":10000,"evals":207,"gcsample":false,"seconds":5.0,"overhead":0.0,"memory_tolerance":0.01}]},"tags":[]}],"16bit":["BenchmarkGroup",{"data":{"Real1D":["Parameters",{"gctrial":true,"time_tolerance":0.05,"samples":10000,"evals":996,"gcsample":false,"seconds":5.0,"overhead":0.0,"memory_tolerance":0.01}],"Mom1D":["Parameters",{"gctrial":true,"time_tolerance":0.05,"samples":10000,"evals":935,"gcsample":false,"seconds":5.0,"overhead":0.0,"memory_tolerance":0.01}]},"tags":[]}]},"tags":[]}]},"tags":[]}],"bose_hubbard_mom_1d_2c":["BenchmarkGroup",{"data":{"small":["Parameters",{"gctrial":true,"time_tolerance":0.05,"samples":3,"evals":1,"gcsample":false,"seconds":100.0,"overhead":0.0,"memory_tolerance":0.01}],"big":["Parameters",{"gctrial":true,"time_tolerance":0.05,"samples":3,"evals":1,"gcsample":false,"seconds":100.0,"overhead":0.0,"memory_tolerance":0.01}]},"tags":[]}]},"tags":[]}]]]
```

Comment:
Is this machine generated? It looks weird.

Reply:
Yes. That's the tunning file. Having it in the repo prevents BenchmarkTools from retunning the benchmarks every time they run.